README.md

```diff
@@ -0,0 +1,65 @@
+# Letter Generator for datenanfragen .de
+
+> This repository contains the source code for Letters Generation (plain text or PDF) from templates  
+
+## Installation
+
+You can install this tools via npm:  
+
+```sh
+npm i letter-generator
+```
+
+## Usage  
+
+You'll have to pass an object to the letter constructor containing those informations:  
+
+```js
+var props = {
+    sender_address: // string | string[]  
```

Comment:
I would prefer to have this be valid JS. It's probably best to include examples for each prop and then have the explanations in the line above, like this:

```
// The sender address, either as a string or as an array of the individual lines.
sender_address: ['Jane Doe', '123 Some Lane', '12345 Some City', 'Some Country'],